srcStatic/ResourceSearchPath.cpp

```diff
@@ -12,7 +12,7 @@ class ResManager {
 	bool GetFilePath(const char* resourceName, /* [out] */ char* filePath) const;
 };
 
-
+// All paths must be an absolute directory
```

Comment:
Hmm, the `ResourceSearchPath` code has no technical requirement for absolute paths. This does seem to get to the root of the problem though, in that relative paths are relative to the current working directory, and depending on how the game was started, that might not be the executable folder.

I'd recommend adding a bit more detail to this comment.